src/main/java/cropcert/certification/ApplicationConfig.java

```diff
@@ -107,26 +109,25 @@ public void onStartup(Container container) {
 
 			@Override
 			public void onShutdown(Container container) {
-
+				// Shutdown the Container
 			}
 
 			@Override
 			public void onReload(Container container) {
-
+				// Reload the Container
 			}
 		});
 		singletons.add(new InterceptorModule());
 
 		return singletons;
 	}
 
-	protected List<Class<?>> getSwaggerAnnotationClassesFromPackage(String packageName)
+	public static List<Class<?>> getSwaggerAnnotationClassesFromPackage(String packageName)
```

Comment:
This change is not neccessary, please revert this back to protected (as present in other modules)